### DISCUSSION TITLE
[Fall 2024] Week 2 Assignment

### DISCUSSION THREAD

**Discussion by jooohyunpark** [jooohyunpark/itp-canvas-for-coders#4] [Assignments]
1. Write a statement for the final project and start developing ideas.
2. Start building scaffold for your final project. Experiment with what we have covered in class.

**Comment by Joyceyyzheng** [comment]
[Isabel & Joyce's Project Idea](https://isabelprwu.notion.site/Canvas-Wk2-Project-Proposal-106a3e40cd34804bafceeb231fb5c0ce?pvs=4)

**Comment by ZoeyFerguson0** [comment]
["Thought Clouds" project idea](https://zoeyferguson.notion.site/Cloud-Portfolio-three-js-106f8e5d9612800aa620c93f368b86d7?pvs=4)

**Comment by michalshoshannyu** [comment]
[A group stares at a glowing object](https://michals-blog-site.webflow.io/weeks/cfc-week3)

**Comment by wallitah** [comment]
[3(d) Studies of Human](https://wallismb.notion.site/3-d-Studies-of-Human-3c2d52636b4044daa45d049cca6c35ee?pvs=4)

**Comment by lqiuuu** [comment]
[ideas](https://lejingqiu.notion.site/Final-project-idea-106ed6a177b0801baee7f7d953d0dce5?pvs=4)

**Comment by YiyangGuo08** [comment]
https://www.notion.so/Final-Project-Proposal-a3d67ac7c2a245d799e7ff978b90b021

**Reply by jooohyunpark** [reply]
Can you make the link public?

**Comment by hds415** [comment]
[Christie's Project Idea](https://www.notion.so/Final-Project-Idea-106f38bad1b4809085ead0b2b4c15d05?pvs=4)

**Reply by jooohyunpark** [reply]
Public link please!

**Reply by hds415** [reply]
https://lowly-purchase-487.notion.site/Final-Project-Idea-106f38bad1b4809085ead0b2b4c15d05?pvs=4

**Comment by edwardszhou** [comment]
Inspiration from [what if?](https://what-if.xkcd.com/34/). grinding away mountains of eternity through type interaction

**Comment by Liming802** [comment]
[Ming](https://dynamic-glider-414.notion.site/Canvas-for-cod-Head-10610edb91888094b05dfeef5b498bf2?pvs=4)

**Comment by murphymuqingw** [comment]
Here is my project idea: [redesign my own portfolio website's landing page](https://zesty-femur-715.notion.site/Project-Redesign-my-own-portfolio-11bd0a2f0b458034a66dcd45729cc449?pvs=4).
